### ISSUE TITLE
Table content filter : visible by default?

### ISSUE THREAD

**Issue by gboudreau** [sequelpro/sequelpro#3084]
I think the table content filter view should be visible by default (same as before - in 1.1.2), or at the very least, there should be a setting in the preferences, to indicate if you prefer to have it open or not by default.

**Comment by gboudreau** [comment]
Closing this, since the new filters pane is not compatible with "empty filter = no filters" that was available in 1.1.2, and thus allowed the filters bar to always be visible.
Ref: #3083

**Comment by dmoagx** [comment]
I don't think that #3083 and this one are mutually exclusive.
After all, if you just open the filter without pressing on "Filter" once it won't get applied currently, either.

**Comment by robertoschwald** [comment]
Need it badly. It is annoying that I have to open the filter content pane every time manually.

**Comment by robertoschwald** [comment]
There'se already a PR to re-enable by default.
https://github.com/sequelpro/sequelpro/pull/3209

**Comment by gboudreau** [comment]
I agree. And that PR, I think, uses the right approach to detect if no filters should be applied (empty values).
Having the filter UI always visible is much more useful to a lot of people versus having the ability to search for empty strings...
Searching for empty string is already an option in the filters selector:
![image](https://user-images.githubusercontent.com/370329/47788601-92e7fa80-dce8-11e8-9b14-91d9b7ae4673.png)

**Comment by dmoagx** [comment]
> Searching for empty string is already an option in the filters selector:

You are reversing cause and issue there.
"is empty" only exists, because there was no other way to express `field = ''` with the previous UI.

One thing I have been thinking about it adding checkboxes in front of the filter rows, so it becomes possible to disable/enable criteria. We could add the first row as "disabled" and enable it, once the user changes it or starts typing.

**Comment by eSkiSo** [comment]
+1 on this, I always had filter open for fast filters (a bit annoying have to keep clicking the icon on the bottom) and then I could just delete the text and table would show all results, now just gives empty query error.

**Comment by NickBourque** [comment]
> You are reversing cause and issue there.
> "is empty" only exists, because there was no other way to express `field = ''` with the previous UI.
> 
@dmoagx not true, you could always just enter field = space with the previous UI. It returned any empty string, regardless of length.

**Comment by chrisjimallen** [comment]
Googled this as it was becoming a pain and IMO unnecessary to keep clicking the filter icon. I think it should be visible by default, or even just the option to pin it.

**Comment by Robert-Saiter** [comment]
> Googled this as it was becoming a pain and IMO unnecessary to keep clicking the filter icon. I think it should be visible by default, or even just the option to pin it.

This project sadly is not being worked on but it was forked to Sequel-Ace which you can find in the App Store.  
https://github.com/Sequel-Ace/Sequel-Ace
